bigtable/google/cloud/bigtable/instance.py

```diff
@@ -220,6 +220,7 @@ def create(self, location_id=_EXISTING_INSTANCE_LOCATION_ID,
         :returns: The long-running operation corresponding to the create
                     operation.
         """
+
         clusters = {}
         cluster_id = '{}-cluster'.format(self.instance_id)
```

Comment:
Please create a `Cluster` object with all of these variables, and add it to an array of clusters, and use that array instead of having duplicate code.

Reply:
Done, thanks.